src/jest.ts

```diff
@@ -59,44 +64,24 @@ export class Jest {
   public readonly config: any;
 
   private readonly ignorePatterns: string[];
+  private readonly project: NodeProject;
 
   constructor(project: NodeProject, options: JestOptions = { }) {
-    const version = options.jestVersion ?? DEFAULT_JEST_VERSION;
+    this.project = project;
 
-    project.addDevDependencies({ jest: version });
+    const version = options.jestVersion ?? DEFAULT_JEST_VERSION;
+    project.addDevDeps(`jest@${version}`);
```

Comment:
I think that with the new "deps" mechanism we don't really need to specify a version by default:

```suggestion
    const version = options.jestVersion;
    const jestDep = version ? `jest@${version}` : 'jest';
    project.addDevDeps(jestDep);
```

This will have the effect of installing the latest jest version when the project is first created and then upgrades can be managed through `yarn upgrade`, dependabot, or any other tool